internal/indexnode/task.go

```diff
@@ -217,6 +217,12 @@ func (it *IndexBuildTask) saveIndexMeta(ctx context.Context) error {
 			panic(errMsg)
```

Comment:
question，why we panic if loadIndexMeta fail?

Reply:
Load failure means etcd is not serviceable.